DN1/test/runtests.jl

```diff
@@ -0,0 +1,35 @@
+using DN1
+using Test
+
+@testset "nakljucno izbrane tocke" begin
```

Comment:
Niso ravno naključne, ampak ste jih izbrali vi.
```suggestion
@testset "Zlpek na 6 točkah" begin
```